### PR TITLE
React 16.13

### PR DESCRIPTION
**Pull request by Marc-Andre-Rivet**
Companion to https://github.com/plotly/dash/pull/1145

### CODE REVIEW

**Comment by emmanuelle** [comment]
thanks for your pull request! The PR is still a draft, do you plan to make more changes?

**Comment by Marc-Andre-Rivet** [comment]
@emmanuelle It's still a draft because I need to make sure it's not causing problems in the other component repos. If you look up the companion PR you'll see that there is 6 other PRs that all need to pass in order to wrap this up :)

File: package.json [hunk 1]
```diff
@@ -23,13 +23,13 @@
...
     "dependencies": {
-        "ramda": "^0.25.0",
-        "react": "15.4.2",
-        "react-dom": "15.4.2",
-        "react-sketch": ">=0.4.4",
         "material-ui": "^0.20.0",
-        "react-color": "^2.13.8",
-        "plotly-icons": ">=1.0"
+        "plotly-icons": ">=1.0",
+        "ramda": "^0.25.0",
```
Comment (alexcjohnson) [inline package.json]: I was going to suggest updating ramda, but it doesn't seem to be used here at all, can we just 🔪 ?

**Comment by emmanuelle** [comment]
@alexcjohnson Ignorance is bliss :-).

File: package.json (hunk 1 above)
Comment (Marc-Andre-Rivet) [reply package.json]: https://github.com/plotly/dash-canvas/pull/47/commits/dd843afd410e60eabb7f59d93bddb71a191315b5

**Pull request merged**